src/components/Letters.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import Letter from './Letter';
+
+class Letters extends Component {
+  render() {
+    const { letters } = this.props;
+    return Object.keys(letters).map((l) =>
+      letters[l] ? (
+        <Letter
+        key={l}
+          selectLetter={this.props.selectLetter}
+          status="checked"
+          letter={l}
+        />
+      ) : (
+        <Letter 
+        key={l}
+          selectLetter={this.props.selectLetter}
+          status="unChecked"
+          letter={l}
```

Comment:
instead of returning the whole <Letter> component twice (when false and when true), you could do something like this:
return Object.keys(letters).map((l) =>
        <Letter
        key={l}
          selectLetter={this.props.selectLetter}
          status={  letters[l] ? "checked" : "unChecked")
          letter={l}
        />

notine that i just put the ternary inside the "status". property that being passed to Letter

Reply:
thanks for the comments, 
i changed it